testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -221,6 +221,10 @@ def test_normal(self, vec1, vec2, value):
         assert_equal(mdamath.normal(vec1, vec2), value)
         # add more non-trivial tests
 
+    @pytest.mark.parametrize('a', np.array([0.1, 0, 0.2]))
+    def test_angle(self, a):
+    	assert_equal(mdamath.angle(a, -a), mdamath.angle(a**0.5, -(a**0.5)))
```

Comment:
Why are you using `parametrize` here? It turns `a` into the scalars `0.1`, `0`, and `0.2` sequentially. Does that test what you want to test? Does comparing the angle between `(a, -a)` and `(a**0.5, -(a**0.5))` test your fix, i.e. that when `np.dot(a, b) / (norm(a) * norm(b)) < -1.0`, `angle()` still returns `pi`?

Note: I specifically used the vector `[0.1, 0, 0.2] ** 0.5` in the issue because it [takes advantage of floating point imprecision](https://medium.com/better-programming/why-is-0-1-0-2-not-equal-to-0-3-in-most-programming-languages-99432310d476).

Reply:
@lilyminium , that article was helpful!
Thanks :)